packages/yourloops/components/patient-list/custom-cells.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FunctionComponent } from 'react'
+import Box from '@mui/material/Box'
+import ReportProblemIcon from '@mui/icons-material/ReportProblem'
+import { useAuth } from '../../lib/auth'
+import { useTheme } from '@mui/material/styles'
+import EmailIcon from '@mui/icons-material/Email'
+import EmailOpenIcon from '../icons/email-open-icon'
+import Tooltip from '@mui/material/Tooltip'
+import { useTranslation } from 'react-i18next'
+import IconActionButton from '../buttons/icon-action'
+import PersonRemoveIcon from '../icons/person-remove-icon'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import FlagIcon from '@mui/icons-material/Flag'
+import FlagOutlineIcon from '@mui/icons-material/FlagOutlined'
+
+interface FlagCellProps {
+  isFlagged: boolean
+  patientId: string
+}
+
+interface AlarmPercentageCellProps {
+  isAlarmActive: boolean
+  value: number
+}
+
+interface MessageCellProps {
+  hasNewMessages: boolean
+}
+
+interface ActionsCellProps {
+  patientId: string
+  onClickRemove: (patientId: string) => void
+}
+
+export const FlagIconCell: FunctionComponent<FlagCellProps> = ({ isFlagged, patientId }) => {
+  const { flagPatient } = useAuth()
+
+  const onClickFlag = async (): Promise<void> => {
+    await flagPatient(patientId)
+  }
+
+  return (
+    <IconActionButton
+      icon={isFlagged
+        ? <FlagIcon
+          titleAccess="flag-icon-active"
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏`title` and `aria-label` must be human-readable strings (and translated) such as `Flag patient toto` or `Unflag patient toto`